api/data_refinery_api/settings.py

```diff
@@ -339,3 +339,6 @@
 
 # EngagementBot
 ENGAGEMENTBOT_WEBHOOK = get_env_variable_gracefully("ENGAGEMENTBOT_WEBHOOK")
+
+# For testing purposes, sometimes we do not want to dispatch jobs unless specifically told to
+DO_NOT_AUTO_DISPATCH = get_env_variable_gracefully("DO_NOT_AUTO_DISPATCH") == "True"
```

Comment:
Could we rename this var `AUTO_DISPATCH_NOMAD_JOBS` to both make it more specific and also to not making it a negative? Telling someone to set `DO_NO_AUTO_DISPATCH` to false is more confusing than telling them to set `AUTO_DISPATCH_JOBS` to true.